app/views/planning_applications/recommendation_form.html.erb

```diff
@@ -21,38 +23,43 @@
         <% end %>
       <% end %>
      <div class="govuk-form-group">
-      <div class="govuk-radios" data-module="govuk-radios" style="margin-bottom: 20px;">
+      <div class="govuk-radios" data-module="govuk-radios">
         <div class="govuk-radios govuk-radios--small">
           <div class="govuk-radios__item">
             <%= form.radio_button :decision, "granted", class: "govuk-radios__input", id:"status-granted-conditional", "aria-controls": "conditional-status-granted-conditional", "aria-expanded": "false" %>
             <%= form.label :decision_granted, "Yes", class: "govuk-label govuk-radios__label", for: "status-granted-conditional" %>
           </div>
           <div class="govuk-radios__item">
             <%= form.radio_button :decision, "refused", class: "govuk-radios__input", id: "status-refused-conditional", "aria-controls": "conditional-status-refused-conditional", "aria-expanded": "false" %>
-            <%= form.label :decision_refused, "No", class: "govuk-label govuk-radios__label govuk-!-margin-bottom-8", for: "status-refused-conditional" %>
+            <%= form.label :decision_refused, "No", class: "govuk-label govuk-radios__label govuk-!-margin-bottom-3", for: "status-refused-conditional" %>
           </div>
         </div>
       </div>
      </div>
       <div class="govuk-form-group">
-        <p class="govuk-body">
-          <%= form.label :public_comment, "Please provide supporting information to indicate which GDPO policy(s) have or have not been met.", class: "govuk-label govuk-!-font-weight-bold" %>
-        </p>
-        <p class="govuk-body">
+        <span class="govuk-body">
+          <%= form.label :public_comment, "State the reasons why this application is, or is not lawful.", class: "govuk-label govuk-!-font-weight-bold" %>
+        </span>
+        <span class="govuk-body">
+          Refer to the specific permitted development rights being evoked.
+          </span>
+        <p class="govuk-hint">
           This information <strong>will</strong> appear on the decision notice.
         </p>
-        <%= form.text_area :public_comment, class: "govuk-textarea govuk-!-margin-bottom-9", rows: "3" %>
+        <%= form.text_area :public_comment, class: "govuk-textarea", rows: "3" %>
       </div>
     </div>
     <div class="govuk-form-group">
-      <p class="govuk-body">
+      <span class="govuk-body">
```

Comment:
What's the reason for making these spans? `p` feels a bit more semantic?

Reply:
It was more to do with the spacing - Ali needed some elements to look like they were sitting with other elements rather than being spaced separately into paragraphs. Using a span meant that we didn't have to use a <p> tag with a whole lot of messy custom minus padding.